gapic-libraries-bom/pom.xml

```diff
@@ -1022,6 +1029,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-gapic-bom</artifactId>
```

Comment:
@suztomo Do you know why this gets inserted here when running the new client generation script?

Reply:
No, I don’t. I don’t recognize this BOM. Finding corresponding commit (and commit message) will clarify which step adds this.